test/acceptance/tests/basic/basic_test.go

```diff
@@ -19,27 +19,12 @@ import (
 	"github.com/gruntwork-io/terratest/modules/random"
 	"github.com/gruntwork-io/terratest/modules/shell"
 	"github.com/gruntwork-io/terratest/modules/terraform"
-	"github.com/hashicorp/consul/sdk/testutil/retry"
+	"github.com/hashicorp/serf/testutil/retry"
```

Comment:
Make sure to use testutils from the consul repo (not serf).

```suggestion
	"github.com/hashicorp/consul/sdk/testutil/retry"
```

My editor magic also likes to switch this out for whatever reason.